MaxText/configs/base.yml

```diff
@@ -121,6 +121,15 @@ record_internal_nn_metrics: 0
 # Create a GCS bucket, e.g. my-maxtext-outputs and set this to "gs://my-maxtext-outputs/"
 base_output_directory: ""
 
+# Whether or not to enable emergency checkpoint. If True, `enable_emergency_checkpoint` must also be specified.
+enable_emergency_checkpoint: False
+
+# It should be speficied when and only when `enable_emergency_checkpoint` is True.
+local_checkpoint_directory: ""
+
+# It should be non-zero when and only when `enable_emergency_checkpoint` is True.
+local_checkpoint_period: 0
```

Comment:
Can we explain what an emergency checkpoint is (maybe in the top comment for enable_emergency_checkpoint)

I am surprised to see a field called "local_checkpoint_period" for emergency checkpointing - I thought the emergency checkpoint would only be saved once. Is this local_checkpoint_period additionally saving it every `period` steps because we are also removing the regular checkpoint manager?

Reply:
Will add more comments. You are right, "emergency checkpointing" is just how Orbax calls the "local checkpointing" feature. We could make it for emergency only, but right now it saves to local disk or RAM Disk periodically.

We are not removing the regular checkpoint manager - it will be part of the ECM and still save to GCS periodically.